packages/cx/src/widgets/List.scss

```diff
@@ -75,11 +75,7 @@
    }
 
    .#{$element}#{$name}-empty-text {
-      margin: 0;
-      padding: 5px 10px;
-
-      color: gray;
-      font-size: smaller;
+      padding: $cx-default-list-item-padding;
```

Comment:
We should use a different color for empty text like we do with inputs and grid controls.

Reply:
color: $cx-default-empty-text-color; 
added to empty text list.